packages/database/prisma/migrations/20260817000000_add_transaction_review_fields/migration.sql

```
@@ -0,0 +1,14 @@
+-- AlterTable
+ALTER TABLE "Transaction" ADD COLUMN "flagReviewAction" TEXT;
+ALTER TABLE "Transaction" ADD COLUMN "resolvedBy" TEXT;
+ALTER TABLE "Transaction" ADD COLUMN "resolvedAt" TIMESTAMP(3);
+ALTER TABLE "Transaction" ADD COLUMN "resolutionNote" TEXT;
+
+-- AlterTable
+ALTER TABLE "ComplianceAlert" ADD COLUMN "ruleId" TEXT;
+
+-- CreateIndex
+CREATE INDEX "Transaction_flagReviewAction_idx" ON "Transaction"("flagReviewAction");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)